classes/class-p4-campaign-importer.php

```diff
@@ -281,5 +282,26 @@ public function process_campaign_metas( $post_meta ) {
 
 			return $post_meta;
 		}
+
+		/**
+		 * Skip already existing postmeta data from import.
+		 *
+		 * @param array   $post_meta The to be imported post meta fields.
+		 * @param integer $post_id Post ID.
+		 *
+		 * @return array The cleaned post meta fields.
+		 */
+		public function skip_duplicate_postmeta_import( $post_meta, $post_id ) {
+			$campaign_postmeta = get_post_meta( $post_id );
+			if ( $post_meta && $campaign_postmeta ) {
```

Comment:
Both of these are variables meta of a campaign, the difference is one exists in db already. Maybe you can rename `$campaign_postmeta` to `$existing_postmeta`.